_posts/2023-02-07-release.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Release 2023.07
+author: Priya
+date: 2023-02-11
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Support for Stored Procedures in Salesforce
+Mammoth now enables users to execute queries using stored procedures in the query box.
+
+#### Improved messages in Add Data modal.
+Mammoth has enhanced the messaging for the components like My computer, Webhooks, etc under Add Data modal.
+
+#### Enhanced Explore card performance
+Mammoth has significantly improved its performance when opening multiple Explore Cards. Opening more than 50 Explore Cards should now take no longer than two minutes.
```

Comment:
Remove "Opening more than 50 Explore Cards should now take no longer than two minutes."

Reply:
fixed